packages/clay-form/src/DualListBox.tsx

```diff
@@ -65,6 +65,16 @@ interface IProps extends React.HTMLAttributes<HTMLDivElement> {
 	 */
 	items: TItems;
 
+	/**
+	 * The maximum number of items for left Select Box.
+	 */
+	leftMaxItems?: number;
```

Comment:
Instead of using `left` and `right` can we use something friendlier for rtl devs like `first`/`last` or `start`/`end`?

Reply:
Makes sense Pat! I'll change.

Reply:
It would have been nice to establish a pattern like this when the component was first made, but not now.

Like [I said](https://github.com/liferay-frontend/liferay-portal/pull/98#pullrequestreview-446252517) on the related DXP [PR](https://github.com/liferay-frontend/liferay-portal/pull/98), the component already has "left"/"right" terminology, so introducing a *mix* of "left"/"right" terminology with "first"/"last" or "start"/"end" terminology would cause confusion (and if we want to change to all, to be consistent, we have an undesirable breaking change).